person_capture/gui_app.py

```
@@ -10722,6 +10722,201 @@ def _save_hdr_archive_crop(
+        if not ref_hdr_path or not os.path.exists(ref_hdr_path):
+            return False, "windows_wic_unavailable:ref_hdr_source_missing", None, None
+        if not clean_hdr_path or not os.path.exists(clean_hdr_path):
+            return False, "windows_wic_unavailable:clean_hdr_source_missing", None, None
+
+        ref_tmp = ref_out_path + ".tmp.raw"
+        clean_tmp = clean_out_path + ".tmp.raw"
+        for out_path in (ref_out_path, clean_out_path):
+            try:
+                ensure_dir(os.path.dirname(out_path))
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)